src/foam/nanos/u2/navigation/TopNavigation.js

```diff
@@ -101,24 +101,33 @@ foam.CLASS({
     {
       name: 'dao',
       factory: () => this.menuDAO
-    }
+    },
+    'welcome'
   ],
 
   methods: [
     function initE() {
       this
         .addClass(this.myClass())
+        .start('div', null, this.welcome$)
+          .add('Welcome').addClass('welcome-label').hide(this.loginSuccess$)
```

Comment:
Let's make this 'Welcome' string into a messages prop.

Reply:
Done.